src/Symfony/Component/Polyfill/bootstrap.php

```diff
@@ -0,0 +1,158 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Symfony\Component\Polyfill as p;
+
+/**
+ * @author Nicolas Grekas <p@tchwork.com>
+ */
+
+if (PHP_VERSION_ID < 50400) {
+    if (!function_exists('trait_exists')) {
+        function trait_exists($class, $autoload = true) {return $autoload && class_exists($class, $autoload) && false;}
+    }
+    if (!function_exists('class_uses')) {
+        function class_uses($class, $autoload = true) {return $autoload && class_exists($class, $autoload) && false;}
+    }
+    if (!function_exists('hex2bin')) {
+        function hex2bin($data) {return p\Php54::hex2bin($data);}
+    }
+}
+
+if (PHP_VERSION_ID < 50500) {
+    if (!function_exists('boolval')) {
+        function boolval($val) {return p\Php55::boolval($val);}
+    }
+    if (!function_exists('json_last_error_msg')) {
+        function json_last_error_msg() {return p\Php55::json_last_error_msg();}
+    }
+    if (!function_exists('array_column')) {
+        function array_column($array, $columnKey, $indexKey = null) {return p\Php55ArrayColumn::array_column($array, $columnKey, $indexKey);}
+    }
+    if (!function_exists('hash_pbkdf2')) {
+        function hash_pbkdf2($algorithm, $password, $salt, $iterations, $length = 0, $rawOutput = false) {return p\Php55::hash_pbkdf2($algorithm, $password, $salt, $iterations, $length, $rawOutput);}
+    }
+}
+
+if (PHP_VERSION_ID < 50600) {
+    if (!function_exists('hash_equals')) {
+        function hash_equals($knownString, $userInput) {return p\Php56::hash_equals($knownString, $userInput);}
+    }
+    if (!function_exists('ldap_escape')) {
```

Comment:
We should not only check whether the `ldap_escape` function exists or not, but also check whether the extension is enabled or not.